gooddata-fdw/gooddata_fdw/import_workspace.py

```diff
@@ -63,15 +63,15 @@ def import_tables(self) -> list[TableDefinition]:
 
 
 class WorkspaceImportersLocator:
-    _IMPORTERS: set[Type[WorkspaceImporter]] = set()
+    _IMPORTERS: set[type[WorkspaceImporter]] = set()
 
     @classmethod
-    def register(cls, class_: Type[WorkspaceImporter]) -> Type[WorkspaceImporter]:
+    def register(cls, class_: type[WorkspaceImporter]) -> type[WorkspaceImporter]:
```

Comment:
This typing is imprecise. This means you pass it _some_ `type[WorkspaceImporter]` and it returns (possibly some other) `type[WorkspaceImporter]`.

The register method should be generic on this with a TypeVar bound to `type[WorkspaceImporter]`.

Reply:
Not super critical though, it does not make the status quo worse ;)

Reply:
Agree. gooddata_fdw is currently marked as deprecated. I propose to keep it as is.